react/components/form/FieldTemplate.js

```diff
@@ -4,17 +4,19 @@ import React from 'react'
 const FieldTemplate = ({
   children,
   classNames,
+  formContext,
   hidden,
+  schema,
 }) => {
-  if (hidden) {
-    return children
+  const isHidden =
+    hidden ||
+    (schema.type !== 'object' && !!schema.isLayout !== formContext.isLayoutMode)
```

Comment:
what's the difference between `!!schema.isLayout` and `schema.isLayout` ?

Reply:
`!!` makes a type coercion to `boolean`, so even properties without an `isLayout` property — i.e., `isLayout` is `undefined` — will return a `boolean` value, and the `!==` inequality operator will work as desired.